config/cscs.py

```diff
@@ -289,6 +289,24 @@
                     'descr': 'JupyterHub multicore nodes',
                     'max_jobs': 10,
                     'launcher': 'srun'
+                },
+                {
+                    'name': 'xfer',
+                    'scheduler': 'slurm',
+                    'environs': [
+                        'builtin',
+                        'PrgEnv-cray',
+                        'PrgEnv-gnu',
+                        'PrgEnv-intel',
+                        'PrgEnv-pgi'
```

Comment:
Do we really need all these environments?

Reply:
No, actually now that you mention it only `builtin` is available, so I think it's better if the hostname check has the `builtin` programming environment.